genanki/__init__.py

```diff
@@ -189,6 +196,29 @@ def __init__(self, model=None, fields=None, sort_field=None, tags=None, guid=Non
       # guid was defined as a property
       pass
 
+    ## Options ##
```

Comment:
Per above comment, please move this out of the `Note` class and into the `Card` class.